flex/validation/parameter.py

```diff
@@ -51,6 +51,8 @@ def type_cast_parameters(parameter_values, parameter_definitions, context):
             parameter_definition = find_parameter(parameter_definitions, name=key)
         except KeyError:
             continue
+        except ValueError:
+            continue
```

Comment:
Hmm, presumably there's another validator that checks that the parameters list is valid (i.e. doesn't contain duplicates), and if so, do we need to worry about this? 

Probably a good idea to split out those into different exceptions anyway: `NoSuchParameter`, `MultipleParametersFound`.

Also just realized that you can specify `parameters` at Path Item Object level that applies to all operations on a given path - not sure if they are accounted for in this code. I'll add a test and verify.